addons/theme_default/static/src/scss/customize_modal.scss

```diff
@@ -1,18 +1,530 @@
+button.close:after, button.close:before{
```

Comment:
This file should be simplified and reorganized. Try removing the rules that are not needed, the design I saw should not need 500lines of scss :wink: 

Also take care of missing spaces / useless line breaks / etc (and see https://github.com/odoo/odoo/wiki/SCSS-coding-guidelines)

Reply:
Try removing useless levels ;) body / .o_techical_modal / ...